components/org.wso2.carbon.event.simulator.core/src/main/java/org/wso2/carbon/event/simulator/core/exception/CSVFileDeploymentMapper.java

```diff
@@ -19,9 +19,10 @@
 
     @Override
     public Response toResponse(CSVFileDeploymentException e) {
-        return Response.status(Response.Status.BAD_REQUEST).
-                entity(new ResponseMapper(Response.Status.BAD_REQUEST, e.getMessage())).
-                type("application/json").
-                build();
+        return Response.status(Response.Status.BAD_REQUEST)
```

Comment:
create an abstract code and then add that method there. so all the concrete implementations can call that.